packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -53,21 +53,19 @@ export interface Distribution {
 
 export interface Histogram {
   /**
-   * Buckets are implemented using two different array:
-   *  - boundaries contains every boundary (which are upper boundary for each slice)
-   *  - counts contains count of event for each slice
+   * Buckets are implemented using two different arrays:
+   *  - boundaries: contains every finite bucket boundary, which are inclusive lower bounds
+   *  - counts contains event counts for each slice
```

Comment:
Omit previous colon or add here for consistency
```suggestion
   *  - counts: contains event counts for each slice
```

Reply:
Fixed, thanks for catching that!